packages/lab/package.json

```diff
@@ -45,6 +45,7 @@
     "@emotion/css": "^11.11.0",
     "@hitachivantara/uikit-react-core": "^5.28.1",
     "@hitachivantara/uikit-react-icons": "^5.6.7",
+    "@hitachivantara/uikit-react-viz": "^5.5.12",
```

Comment:
We don't want this. We don't need this.
If ESLint complains in the stories files, then it's wrong and we should ignore it